bin/filter.rb

```diff
@@ -56,6 +56,13 @@ def process(data, force_header: nil, format_for_frontpage: false)
   doc.css("strong:nth-child(1)").each do |node|
     id = node.content.downcase.gsub(/[^a-z0-9-]+/, '-').gsub(/^-|-$/, '')
     node["id"] = id if id.length > 1
+
+    if (format_for_frontpage == false)
+      anchor_link = "<a href=\"##{id}\" style=\"position: absolute; left: -20px; color: var(--menu-text); text-decoration: none;\">&para;</a>"
+
+      node.parent[:style] = (node.parent[:style] || "") + "position: relative;"
```

Comment:
This won't work if the parent node's style doesn't end in a `;`, which it might not?

How about:

```ruby
[node.parent[:style], "position: relative;"].compact.join(";").squeeze(";")
```

```
2.6.3 :004 > ["blah: foo;", "position: relative;"].compact.join("; ").squeeze(";")
 => "blah: foo; position: relative;"
2.6.3 :005 > ["blah: foo;;;;", "position: relative;"].compact.join("; ").squeeze(";")
 => "blah: foo; position: relative;"
2.6.3 :006 > ["blah: foo", "position: relative;"].compact.join("; ").squeeze(";")
 => "blah: foo; position: relative;"
2.6.3 :007 > [nil, "position: relative;"].compact.join("; ").squeeze(";")
 => "position: relative;"
```

Reply:
Well that's nice.